packages/lib/theme.ts

```diff
@@ -22,340 +22,330 @@ const themes: any = {
-	}
-
-	if (!output.colorBright2) {
-		output.colorBright2 = output.appearance === 'light' ? '#ffffff' : '#ffffff';
-	}
-
-	return output;
+	return { ...themes[themeId] };
 }
 
+const literal = <T extends string> (str: T): T => str;
+
 // globalStyle should be used for properties that do not change across themes
 // i.e. should not be used for colors
-// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
-const globalStyle: any = {
-	fontFamily: 'Roboto', // 'sans-serif',
-	margin: 15, // No text and no interactive component should be within this margin
-	itemMarginTop: 10,
-	itemMarginBottom: 10,
-	disabledOpacity: 0.3,
-	buttonMinWidth: 50,
-	buttonMinHeight: 30,
-	editorFontSize: 12,
-	textAreaLineHeight: 17,
-	lineHeight: '1.6em',
-	headerButtonHPadding: 6,
-	toolbarHeight: 26,
-	toolbarPadding: 6,
-	appearance: 'light',
-	mainPadding: 12,
-	topRowHeight: 50,
-	editorPaddingLeft: 8,
-};
-
-globalStyle.marginRight = globalStyle.margin;
-globalStyle.marginLeft = globalStyle.margin;
-globalStyle.marginTop = globalStyle.margin;
-globalStyle.marginBottom = globalStyle.margin;
-
-globalStyle.icon = {
-	fontSize: 30,
-};
-
-globalStyle.lineInput = {
-	fontFamily: globalStyle.fontFamily,
-	maxHeight: 22,
-	height: 22,
-	paddingLeft: 5,
-};
-
-globalStyle.headerStyle = {
-	fontFamily: globalStyle.fontFamily,
-};
-
-globalStyle.inputStyle = {
-	border: '1px solid',
-	height: 24,
-	maxHeight: 24,
-	paddingLeft: 5,
-	paddingRight: 5,
-	boxSizing: 'border-box',
-};
-
-globalStyle.containerStyle = {
-	overflow: 'auto',
-	overflowY: 'auto',
-};
-
-globalStyle.buttonStyle = {
-	// marginRight: 10,
-	border: '1px solid',
-	minHeight: 26,
-	minWidth: 80,
-	// maxWidth: 220,
-	paddingLeft: 12,
-	paddingRight: 12,
-	paddingTop: 6,
-	paddingBottom: 6,
-	// boxShadow: '0px 1px 1px rgba(0,0,0,0.3)',
-	fontSize: globalStyle.fontSize,
-	borderRadius: 4,
-};
+const globalStyle = (() => {
+
+	const margin = 15; // No text and no interactive component should be within this margin
+	const fontFamily = 'Roboto'; // 'sans-serif',
+	return {
+		fontFamily: fontFamily,
+		itemMarginTop: 10,
+		itemMarginBottom: 10,
+		disabledOpacity: 0.3,
+		buttonMinWidth: 50,
+		buttonMinHeight: 30,
+		editorFontSize: 12,
+		textAreaLineHeight: 17,
+		lineHeight: '1.6em',
+		headerButtonHPadding: 6,
+		toolbarHeight: 26,
+		toolbarPadding: 6,
+		appearance: ThemeAppearance.Light,
+		mainPadding: 12,
+		topRowHeight: 50,
+		editorPaddingLeft: 8,
+
+		margin: margin,
+		marginRight: margin,
+		marginLeft: margin,
+		marginTop: margin,
+		marginBottom: margin,
+
+		icon: { fontSize: 30 },
+		lineInput: {
+			fontFamily,
+			maxHeight: 22,
+			height: 22,
+			paddingLeft: 5,
+		},
+		headerStyle: {
+			fontFamily,
+		},
+		inputStyle: {
+			border: '1px solid',
+			height: 24,
+			maxHeight: 24,
+			paddingLeft: 5,
+			paddingRight: 5,
+			boxSizing: literal('border-box'),
```

Comment:
See the pull request description for an explanation of this usage of `literal(...)`. In short, it allows existing code to do calculations similar to `theme.inputStyle.height * 2` while also allowing styles to be passed to React (e.g. with `<a style={theme.inputStyle}>...</a>`.